SWLOR.Game.Server/Event/Module/mod_on_load.cs

```diff
@@ -127,6 +127,8 @@ private static void SetModuleEventScripts()
             NWNXEvents.SubscribeEvent(EventType.DecrementItemStackSizeBefore, "item_dec_stack");
             NWNXEvents.SubscribeEvent(EventType.UseItemBefore, "item_use_before");
             NWNXEvents.SubscribeEvent(EventType.UseItemAfter, "item_use_after");
+            NWNXEvents.SubscribeEvent(EventType.ItemInventoryRemoveItemAfter, "item_inventory_disturb_after");
```

Comment:
I think these script names are too long. There's a cap of 16 characters on resrefs aren't there?

Reply:
In NWN yes, but since these are all c# classes it seems to work fine.   I've tested and confirmed that this code correctly gets called as-is.

Reply:
Interesting. Maybe I'm forgetting how scripts get dispatched in legacy. Anyway, cool! Thanks for the clarification.